client/sections.js

```diff
@@ -7,85 +7,124 @@ sections = [
 	{
 		name: 'customize',
 		paths: [ '/customize' ],
-		module: 'my-sites/customize'
+		module: 'my-sites/customize',
+		group: 'sites'
 	},
 	{
 		name: 'post-editor',
 		paths: [ '/post', '/page' ],
-		module: 'post-editor'
+		module: 'post-editor',
+		group: 'editor',
+		secondary: true
 	},
 	{
 		name: 'me',
 		paths: [ '/me', '/purchases' ],
-		module: 'me'
+		module: 'me',
+		group: 'me',
+		secondary: true
 	},
 	{
 		name: 'media',
 		paths: [ '/media' ],
-		module: 'my-sites/media'
+		module: 'my-sites/media',
+		group: 'sites',
+		secondary: true
 	},
 	{
 		name: 'menus',
 		paths: [ '/menus' ],
-		module: 'my-sites/menus'
+		module: 'my-sites/menus',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'people',
 		paths: [ '/people' ],
-		module: 'my-sites/people'
+		module: 'my-sites/people',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'plugins',
 		paths: [ '/plugins' ],
-		module: 'my-sites/plugins'
+		module: 'my-sites/plugins',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'posts-pages',
 		paths: [ '/pages' ],
-		module: 'my-sites/pages'
+		module: 'my-sites/pages',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'posts-pages',
 		paths: [ '/posts' ],
-		module: 'my-sites/posts'
+		module: 'my-sites/posts',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'settings',
 		paths: [ '/settings' ],
-		module: 'my-sites/site-settings'
+		module: 'my-sites/site-settings',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'sharing',
 		paths: [ '/sharing' ],
-		module: 'my-sites/sharing'
+		module: 'my-sites/sharing',
+		secondary: true,
+		group: 'sites'
 	},
 	{
 		name: 'signup',
 		paths: [ '/start', '/phone', '/log-in', '/jetpack' ],
 		module: 'signup',
+		secondary: false,
 		enableLoggedOut: true
 	},
 	{
 		name: 'stats',
 		paths: [ '/stats' ],
-		module: 'my-sites/stats'
+		module: 'my-sites/stats',
+		secondary: true,
+		group: 'sites'
+	},
+	{
+		name: 'theme',
+		paths: [ '/theme' ],
+		module: 'my-sites/themes',
+		enableLoggedOut: config.isEnabled( 'manage/themes/logged-out' ),
+		routing: config( 'env' ) === 'development' ? 'isomorphic' : '',
```

Comment:
Uh, my bad. I must've introduced that with #3562 (though I could've sworn I removed it before merging). Not using this yet at all, but planning to use it with #3302.

Also, :+1: for a proper flag.